tmva/tmva/src/RBDT.cxx

```diff
@@ -1,4 +1,389 @@
-#include "TMVA/RBDT.hxx"
+/**********************************************************************************
+ * Project: ROOT - a Root-integrated toolkit for multivariate data analysis       *
+ * Package: TMVA                                                                  *
+ *                                                                                *
+ *                                                                                *
+ * Description:                                                                   *
+ *                                                                                *
+ * Authors:                                                                       *
+ *      Jonas Rembser (stefan.wunsch@cern.ch)                                     *
+ *                                                                                *
+ * Copyright (c) 2024:                                                            *
+ *      CERN, Switzerland                                                         *
+ *                                                                                *
+ * Redistribution and use in source and binary forms, with or without             *
+ * modification, are permitted according to the terms listed in LICENSE           *
+ * (see tmva/doc/LICENSE)                                          *
+ **********************************************************************************/
 
-template class TMVA::Experimental::RBDT<TMVA::Experimental::BranchlessForest<float>>;
-template class TMVA::Experimental::RBDT<TMVA::Experimental::BranchlessJittedForest<float>>;
+#include <TMVA/RBDT.hxx>
+
+#include <TFile.h>
+
+#include <cmath>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <sstream>
+#include <stdexcept>
+#include <stdlib.h>
+#include <string>
+#include <vector>
+
+namespace {
+
+namespace detail {
+
+typedef std::map<int, int> IndexMap;
```

Comment:
You usually want `unordered_map`s because they have better runtime complexity, unless the code really needs the key ordering guarantees